### ISSUE TITLE
Dependency hell...

### ISSUE THREAD

**Issue by apeltzer** [nf-core/rrna-ampliseq#25]
Had to polish the environmen.yaml quite a bunch of times unfortunately:

- ncurses is requiring 5.9 but doesn#t specify directly from where, prefixing with `conda-forge::ncurses=5.9` solved that issue

Similar things happen then when running `qiime_import`:

```
ampliseq-1.0dev/lib/python3.5/site-packages/matplotlib/backends/qt_editor/figureoptions.py", line 20, in <module>
      import matplotlib.backends.qt_editor.formlayout as formlayout
    File "/opt/conda/envs/nf-core-rrna-ampliseq-1.0dev/lib/python3.5/site-packages/matplotlib/backends/qt_editor/formlayout.py", line 54, in <module>
      from matplotlib.backends.qt_compat import QtGui, QtWidgets, QtCore
    File "/opt/conda/envs/nf-core-rrna-ampliseq-1.0dev/lib/python3.5/site-packages/matplotlib/backends/qt_compat.py", line 140, in <module>
      from PyQt5 import QtCore, QtGui, QtWidgets
  ImportError: libGL.so.1: cannot open shared object file: No such file or directory
  
  An unexpected error has occurred:
  
    libGL.so.1: cannot open shared object file: No such file or directory
  
  See above for debug info.

Work dir:
  /home/alex/IDEA/nf-core/rrna-ampliseq/work/be/06192acf88d8197b617ef9e3f8d064
```

Apparently the qt library installed doesn't install the required `libGL.so.1` shared objects.

**Comment by apeltzer** [comment]
https://github.com/conda-forge/pygridgen-feedstock/issues/10

**Comment by apeltzer** [comment]
Might be that we need to ask them a bit ... 

https://forum.qiime2.org/c/dev-discussion

**Comment by apeltzer** [comment]
https://forum.qiime2.org/t/pyqt-related-errors/847

**Comment by apeltzer** [comment]
I was able to get this done. Apparently, its an issue to run in a docker container and thus in headless mode. Fixing this required to set the matplotlib backend to something else:
 
```
echo "backend: Agg" > ~/.config/matplotlib/matplotlibrc
```

**Comment by apeltzer** [comment]
https://forum.qiime2.org/t/error-in-qiime-feature-table-summarize/308/2?u=thermokarst

**Comment by apeltzer** [comment]
Fixed in latest commit f5eb0d107fd4373c6f87b59b8f8615ac6328ecd2
